_resources/port1.0/group/stack_check_fix-1.0.tcl

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# This PortGroup attempts to work around a bug in Xcode 11 on Catalina.
+# See https://forums.developer.apple.com/thread/121887
+#
+# Usage:
+#
+#   PortGroup        stack_check_fix 1.0
+#
+#   stack_check.fix_type:
+#      append_to_compiler_flags (add -fno-stack-check to compiler flags)
+#      append_to_compiler_name  (add -fno-stack-check to compiler name)
+#      avoid_xcode_compiler     (blacklist broken compiler)
+#
+#   stack_check.fixed_version: minimum developer version in which bug is fixed
+
+PortGroup cltversion                  1.0
+
+namespace eval stack_check {
+}
+
+options stack_check.fix_type
+options stack_check.fixed_version
+
+default stack_check.fix_type {append_to_compiler_flags}
+default stack_check.fixed_version {11.2}
```

Comment:
Would it be better to call this option ‘fixed_xcode_version’ ? To me as it is its unclear what entity version applies to.

Reply:
You make a good point.
However, the version not only applies to the Xcode version but also to the CLT version.

Reply:
maybe 'fixed_sdk_version' then ?